app/models/vote.rb

```diff
@@ -0,0 +1,8 @@
+class Vote < ApplicationRecord
+  belongs_to :user
+  belongs_to :work
+
```

Comment:
I suggest a validation to ensure that the [combination](https://guides.rubyonrails.org/active_record_validations.html#uniqueness) of `work_id` and `user_id` is unique.